nix/packages/grpc-haskell-core.nix

```diff
@@ -7,9 +7,9 @@ mkDerivation {
   pname = "grpc-haskell-core";
   version = "0.5.0";
   src = fetchgit {
-    url = "https://github.com/awakesecurity/gRPC-haskell";
-    sha256 = "0sh553lvz3vi1mq65jicy6n3ga4zcifabvapi217kpivxjsa18g6";
-    rev = "414ae8e6612e8e28d2bcfb6e201303f5fc031e5a";
+    url = "git@github.com:awakesecurity/gRPC-haskell.git";
```

Comment:
Need to use the https version since you want the public checkout rather than the private ssh checkout.

Reply:
Oh okay thanks.